mne/preprocessing/ica.py

```diff
@@ -1370,6 +1371,63 @@ def plot_components(self, picks=None, ch_type=None, res=64, layout=None,
                                    image_interp=image_interp,
                                    head_pos=head_pos)
 
+    def plot_properties(self, inst, picks=None, axes=None, dB=True,
+                        cmap=None, plot_std=True, topo_args=None,
+                        image_args=None, psd_args=None, figsize=None,
+                        show=True):
+        """Display component properties: topography, epochs image, ERP,
+        power spectrum and epoch variance.
+
+        Parameters
+        ----------
+        inst: instance of Epochs or Raw
+            The data to use in plotting properties.
+        picks : int | array-like of int | None.
+            The components to be displayed. If None, plot will show the first
+            five sources in the order as fitted. If more than one components
+            were chosen in the picks - each one will be plotted in a separate
+            figure. Defaults to None.
+        axes: list of matplotlib axes | None
+            List of five matplotlib axes to use in plotting: [topo_axis,
+            image_axis, erp_axis, spectrum_axis, variance_axis]. If None a new
```

Comment:
@agramfort I have called it ts_axis (for time series axis) in plot_joint, but maybe ERP would be better?